### PR TITLE
Done with Q&A and team finalized

### PR DESCRIPTION
**Pull request by Windsvept**
@dunphyben

### CODE REVIEW

File: teams.md [hunk 1]
```diff
@@ -39,11 +39,11 @@ Copy the above code snippet in its entirety and add it to the BOTTOM of this doc
 * [GitHub URL](https://github.com/your-profile/your-project-name)
 ### Project Q&A:
 1. What are you going to build?  
-// your answer here  
+  Making giveaway for creators and users easier and more accessible / propaGATING creators
```
Comment (dunphyben) [inline teams.md:42]: Do not delete our example answers. You should answer these questions in your own section.

File: teams.md [hunk 2]
```diff
@@ -71,16 +71,17 @@ Copy the above code snippet in its entirety and add it to the BOTTOM of this doc
...
     - Jun Sub Choi (@Stard)
     - Andrew Ngo (@AndyNoAwkward)
     - David Mai (@2BiK)
     - Alan Fu (@BrandHeart)
+    - Selena Xu (@SelenaX)
+    - Jacob Castro (@Jacobcastro)
 * Seeking Teammates: No
-* [Project URL](https://github.com/Broke-Bois-Club)
-* Project Name tbd
+* [Project URL](https://github.com/RaffleGator/RaffleGator)
```
Comment (dunphyben) [inline teams.md:84]: Please add the questions/answers below this line.

**Comment by verythorough** [comment]
@Windsvept could you change this PR or make a new one with your team information at the bottom of the page?

(If you're not sure ehow to do that, feel free to come to the Netlify table and I can help you out.

**Comment by Windsvept** [comment]
fixed issues and created new pull request

**Pull request closed without merging**